config/puma.rb

```diff
@@ -1,43 +1,51 @@
-# the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum; this matches the default thread size of Active Record.
-#
-max_threads_count = ENV.fetch("RAILS_MAX_THREADS", 5)
-min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
-threads min_threads_count, max_threads_count
-
-# Specifies the `worker_timeout` threshold that Puma will use to wait before
-# terminating a worker in development environments.
+# This configuration file will be evaluated by Puma. The top-level methods that
+# are invoked here are part of Puma's configuration DSL. For more information
+# about methods provided by the DSL, see https://puma.io/puma/Puma/DSL.html.
 #
-worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
-
-# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
+# Puma starts a configurable number of processes (workers) and each process
+# serves each request in a thread from an internal thread pool.
 #
-port ENV.fetch("PORT", 3000)
+# You can control the number of workers using ENV["WEB_CONCURRENCY"]. You
+# should only set this value when you want to run 2 or more workers. The
+# default is already 1.
+#
+# The ideal number of threads per worker depends both on how much time the
+# application spends waiting for IO operations and on how much you wish to
+# prioritize throughput over latency.
+#
+# As a rule of thumb, increasing the number of threads will increase how much
+# traffic a given process can handle (throughput), but due to CRuby's
+# Global VM Lock (GVL) it has diminishing returns and will degrade the
+# response time (latency) of the application.
+#
+# The default is set to 3 threads as it's deemed a decent compromise between
+# throughput and latency for the average Rails application.
+#
+# Any libraries that use a connection pool or another resource pool should
+# be configured to provide at least as many connections as the number of
+# threads. This includes Active Record's `pool` parameter in `database.yml`.
+threads_count = ENV.fetch("RAILS_MAX_THREADS", 5)
+threads threads_count, threads_count
 
 # Specifies the `environment` that Puma will run in.
 #
 environment ENV.fetch("RAILS_ENV", "development")
 
-# Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE", "tmp/pids/server.pid")
-
-# Specifies the number of `workers` to boot in clustered mode.
-# Workers are forked web server processes. If using threads and workers together
-# the concurrency of the application would be max `threads` * `workers`.
-# Workers do not work on JRuby or Windows (both of which do not support
-# processes).
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments. Defaults to 60 seconds.
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
```

Comment:
other updates used `1.hour.to_i` rather than the second count.  
It makes it easier to read if in a rush

Reply:
This line has just been moved/retained. its the same in LFA. rails app:update removed it and not sure we actually need it - see [worker_timeout](https://www.rubydoc.info/gems/puma/Puma%2FDSL:worker_timeout), default is 60 seconds. 

Making it time would require the activesupport time to be loaded too.

Reply:
@colinbruce we do not have it in Apply and Assure. Perhaps we should just remove and rely on default?

Reply:
Happy either way, dealers choice! 😄

Reply:
Am going to remove here and do the same on LFA and Assure, for consistency